package.json

```diff
@@ -14,11 +15,15 @@
     "watch:integration": "cross-env TZ=UTC NODE_ICU_DATA=node_modules/full-icu jest -c scripts/jest/config.integration.js --watch",
     "lint:types": "tsc",
     "lint:eslint": "eslint packages",
-    "lint": "yarn lint:eslint; yarn lint:types",
+    "lint:all": "yarn lint:eslint; yarn lint:types",
     "release": "node ./scripts/release.js",
     "release:integration": "node ./scripts/integration.js",
     "release:build": "node ./scripts/build/",
     "release:test": "node ./scripts/test.js",
+    "version:next": "lerna version --no-private --preid next --create-release github --conventional-commits --conventional-prerelease --yes",
+    "version:latest": "lerna version --no-private --create-release github --conventional-commits --yes",
```

Comment:
Does Lerna uses semantic-release internally or does it handle similar process on it's own (e.g. generating Changelogs, etc)?

Reply:
Yes does it internally, much better because works out of the box with monorepos.